### PR TITLE
Switch `f.submit` over to `f.govuk_submit` for consistent HTML

### PR DESCRIPTION
**Pull request by peteryates**
The `govuk_submit` form helper ensures the relevant `data-module` attributes are set and double-click prevention is enabled too.

### CODE REVIEW

**Review by frankieroberto — APPROVED** [review]

File: app/views/users/sessions/redirect_from_magic_link.html.erb [hunk 1]
```diff
@@ -6,7 +6,7 @@
       <div class="field">
         <%= hidden_field_tag :login_token, @login_token %>
       </div>
-      <%= submit_tag "Continue", class: "govuk-button" %>
+      <%= tag.button("Continue", class: "govuk-button", data: { module: "button" }) %>
```
Comment (frankieroberto) [inline app/views/users/sessions/redirect_from_magic_link.html.erb:9]: Switch this to `f.govuk_submit` by changing `form_tag` to `form_for` (even if there’s no model object)?

Comment (peteryates) [reply app/views/users/sessions/redirect_from_magic_link.html.erb:9]: I was under the impression `form_for` _needed_ an object to bind to

Comment (peteryates) [reply app/views/users/sessions/redirect_from_magic_link.html.erb:9]: Actually maybe `form_with` is the answer here. Let me test it.

Comment (frankieroberto) [reply app/views/users/sessions/redirect_from_magic_link.html.erb]: > I was under the impression form_for needed an object to bind to

I think that was maybe true once but changed at some point.

**Review by frankieroberto — APPROVED** [review]
Some of the Cypress tests are failing as they’re overly specific. 😔

**Comment by peteryates** [comment]
Ah, I fixed a few but couldn't find a 'run all' button in the client so lazily pushed. Will get it green in the morning.

**Comment by frankieroberto** [comment]
> Ah, I fixed a few but couldn't find a 'run all' button in the client so lazily pushed. Will get it green in the morning.

👍  (longer-term maybe we could consider reducing the number of Cyprus / Chromium tests and rely more on capybara tests?)

**Comment by peteryates** [comment]
Yeah, I've wanted to get rid of them for a long time.

I don't think we really use any custom JavaScript so I'm not sure what they're bringing to the table at the moment.

**Pull request merged**